api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -118,6 +118,18 @@ message Plan {
   // Configuration of the radio device used for uplinking to the satellite. Ground stations will
   // need to configure transmission during the plan to match this device.
   radio.RadioDeviceConfiguration uplink_radio_device = 6;
+
+  // The start time of the plan.
```

Comment:
I think definition of start time should be added in the comment so that GS owner understand difference between AOS and start time.